### ISSUE TITLE
network/network2 API docs empty

### ISSUE THREAD

**Issue by kuba** [letsencrypt/letsencrypt#344]
http://letsencrypt.readthedocs.org/en/latest/api/client/network.html
http://letsencrypt.readthedocs.org/en/latest/api/client/network2.html

**Comment by kuba** [comment]
http://letsencrypt.readthedocs.org/en/latest/api/client/client.html and http://letsencrypt.readthedocs.org/en/latest/api/client/revoker.html are also affected.

https://readthedocs.org/builds/letsencrypt/2561198/:
```
/var/build/user_builds/letsencrypt/checkouts/latest/docs/api/client/client.rst:4: WARNING: autodoc: failed to import module u'letsencrypt.client.client'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/letsencrypt/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 335, in import_object
    __import__(self.modname)
  File "/var/build/user_builds/letsencrypt/checkouts/latest/letsencrypt/client/client.py", line 17, in <module>
    from letsencrypt.client import network
  File "/var/build/user_builds/letsencrypt/checkouts/latest/letsencrypt/client/network.py", line 15, in <module>
    requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()
AttributeError: 'module' object has no attribute 'packages'
/var/build/user_builds/letsencrypt/checkouts/latest/docs/api/client/network.rst:4: WARNING: autodoc: failed to import module u'letsencrypt.client.network'; the following exception was raised:
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/letsencrypt/envs/latest/local/lib/python2.7/site-packages/sphinx/ext/autodoc.py", line 335, in import_object
    __import__(self.modname)
  File "/var/build/user_builds/letsencrypt/checkouts/latest/letsencrypt/client/network.py", line 15, in <module>
    requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()
AttributeError: 'module' object has no attribute 'packages'
```

Interestingly,

```
Using /var/build/user_builds/letsencrypt/envs/latest/lib/python2.7/site-packages
Searching for requests==2.2.1
Best match: requests 2.2.1
requests 2.2.1 is already the active version in easy-install.pth
```

despite ticking off "Give the virtual environment access to the global site-packages dir." at https://readthedocs.org/dashboard/letsencrypt/advanced/...

**Comment by kuba** [comment]
I believe this is fixed as of #604.

- http://letsencryptkuba.readthedocs.org/en/rtfd/pkgs/acme/index.html#module-acme.client
- http://letsencryptkuba.readthedocs.org/en/rtfd/api/client.html
- http://letsencryptkuba.readthedocs.org/en/rtfd/api/revoker.html

**Comment by jdkasten** [comment]
Thank you.
